sjb/config/test_cases/test_branch_origin_extended_conformance_install_containerized.yml

```diff
@@ -70,15 +71,19 @@ extensions:
                          -e containerized=true      \
                          -e deployment_type=origin  \
                          /usr/share/ansible/openshift-ansible/playbooks/byo/openshift-node/network_manager.yml
+        # add extra tag to docker images in a form of v#.#.#.#
+        for image in $(docker images --format "{{.Repository}}:{{.Tag}}:" | grep ":$( cat ./ORIGIN_COMMIT ):" | cut -d':' -f1 | sort -u); do
+                docker tag ${image}:$( cat ./ORIGIN_COMMIT ) ${image}:$( cat ./ORIGIN_TAG )
```

Comment:
Ain't working. That will tag all images with `yourtag` name, not with `${tag}:yourtag`.
Closest thing to do is to override `OS_RELEASE_COMMIT` env. Which is set in `lib/build/archive.sh:143`:
```sh
export OS_RELEASE_COMMIT="$(cat ${OS_OUTPUT_RELEASEPATH}/.commit)"
```

After rewriting the `.commit` file with:
```
cat ../aos-cd-jobs/ORIGIN_TAG > `_output/local/releases/.commit
```

the docker images are properly tagged. I find this minimally hacky. But still, that is something that needs to be fixed in the origin build scripts.